storybookjs/storybook
# viewport-addon: Add support for adding new viewports/devices

Note: This is my first open-source PR.

Issue: Currently you cannot customize or add new viewports/devices as described in the README file

## What I did
* Added support to add more viewports/devices
* Added support to replace the entire viewports
* Organized the code respectively
* Added `viewport` addon to the `cra-kitchen-sink` example

## How to test
1. `cd examples/cra-kitchen-sink`
2. `yarn storybook`
3. You will see the `viewport` panel is added
4. Change the `viewports.json` file locaed under `.storybook`
5. Observe the changes

Is this testable with jest or storyshots?
Yes

Does this need a new example in the kitchen sink apps?
No, I added it to the `cra-kitchen-sink` already

Does this need an update to the documentation?
Yes, but once the PR is reviewed

If your answer is yes to any of these, please make sure to include it in your PR.
